pkg/util/testing/wrappers.go

```diff
@@ -831,6 +831,15 @@ func (rf *ResourceFlavorWrapper) Obj() *kueue.ResourceFlavor {
 	return &rf.ResourceFlavor
 }
 
+// ObjectMetaLabel sets the label on the ResourceFlavor.
+func (rf *ResourceFlavorWrapper) ObjectMetaLabel(k, v string) *ResourceFlavorWrapper {
```

Comment:
```suggestion
func (rf *ResourceFlavorWrapper) Label(k, v string) *ResourceFlavorWrapper {
```
 to have similar naming for similar action as other wrappers

Reply:
Done